config/clusters/cloudbank/csm.values.yaml

```diff
@@ -29,15 +29,28 @@ jupyterhub:
         authenticator_class: cilogon
       CILogonOAuthenticator:
         oauth_callback_url: https://csm.cloudbank.2i2c.cloud/hub/oauth_callback
-        username_claim: email
+        shown_idps:
+          - http://google.com/accounts/o8/id
+          - urn:mace:incommon:berkeley.edu
+        allowed_idps:
+          http://google.com/accounts/o8/id:
+            username_derivation:
+              username_claim: "email"
+            allowed_domains:
```

Comment:
Is the preferred method for listing allowed domains a yaml list or an array-like listing["....", "..."]?

Reply:
Either works. I started with with the square brackets when there were around 3 domains and switched to having them one a row when they seemed to many or to long to fit on the same row.